gandi/cli/commands/domain.py

```diff
@@ -38,6 +38,18 @@ def info(gandi, resource):
 
 
 @cli.command()
+@click.argument('resource', metavar='DOMAIN')
+@pass_gandi
+def check(gandi, resource):
+    """Check domain availability and price."""
+
+    result_check = gandi.domain.check(resource)
+    output_domain_check(gandi, result_check, justify=12)
```

Comment:
you can remove the `justify` parameter from the call as it's the same as the method default one.